lib/transaction.rb

```diff
@@ -0,0 +1,20 @@
+class Transaction
+  attr_accessor :id,
+              :invoice_id,
+              :credit_card_number,
+              :credit_card_expiration_date,
+              :result,
+              :created_at,
+              :updated_at
+
+  def initialize(data)
+    @id = data[:id]
+    @invoice_id = data[:invoice_id]
+    @credit_card_number = data[:credit_card_number]
+    @credit_card_expiration_date = data[:credit_card_expiration_date]
+    @result = data[:result]
+    @created_at = Time.now
```

Comment:
for initialization @created_at = data[:created_at] same for updated_at. These values are populated in the csv